profiles/video.jsonld

```diff
@@ -0,0 +1,551 @@
+          "rule": "recommended"
+        }
+      ]
+    },
+    {
+      "id": "https://w3id.org/xapi/video/templates#basic",
+      "name": {
+        "en": "Video Interactions"
+      },
+      "objectActivityType": "https://w3id.org/xapi/video/activity-type/video",
+      "rules": [{
+        "location": "verb.id",
+        "any": [
+          "https://w3id.org/xapi/video/verbs/played",
+          "https://w3id.org/xapi/video/verbs/paused",
+          "https://w3id.org/xapi/video/verbs/seeked"
+        ]
+        }]
+      }
+  ],
+  "patterns": [
+      {
+         "id": "_:generalpattern",
+         "primary": true,
+          "name": {
+              "en": "General Pattern"
+          },
+          "definition": {
+              "en": "This pattern describes the overall sequence of statements sent over the course of an attempt"
+          },
+          "sequence": ["https://w3id.org/xapi/video/templates#initialized", "_:optionalmiddlestatements", "https://w3id.org/xapi/video/templates#terminated"]
+      },
+      {
+          "@id": "_:played-pattern",
+          "alternates": ["https://w3id.org/xapi/video/templates#generalrestrictions", "https://w3id.org/xapi/video/templates#played"]
+      },
+      {
+          "@id": "_:played-completed-pattern",
+          "alternates": ["https://w3id.org/xapi/video/templates#generalrestrictions", "https://w3id.org/xapi/video/templates#played", "https://w3id.org/xapi/video/templates#completed"]
+      },
+      {
+          "@id": "_:played-paused-completed-pattern",
+          "alternates": ["https://w3id.org/xapi/video/templates#generalrestrictions", "https://w3id.org/xapi/video/templates#played", "https://w3id.org/xapi/video/templates#paused", "https://w3id.org/xapi/video/templates#completed"]
+      },
+      {
+          "@id": "_:played-paused-seeked-completed-pattern",
+          "alternates": ["_:generalrestrictions", "https://w3id.org/xapi/video/templates#played", "https://w3id.org/xapi/video/templates#paused", "https://w3id.org/xapi/video/templates#seeked", "https://w3id.org/xapi/video/templates#completed"]
+      },
+      {
+          "@id": "_:played-paused-seeked-volumechange-completed-pattern",
+          "alternates": ["https://w3id.org/xapi/video/templates#generalrestrictions", "https://w3id.org/xapi/video/templates#played", "https://w3id.org/xapi/video/templates#paused", "https://w3id.org/xapi/video/templates#seeked", "https://w3id.org/xapi/video/templates#volumechange", "https://w3id.org/xapi/video/templates#completed"]
+      },
+      {
+          "@id": "_:all-activities-pattern",
+          "alternates": ["https://w3id.org/xapi/video/templates#generalrestrictions", "https://w3id.org/xapi/video/templates#played", "https://w3id.org/xapi/video/templates#paused", "https://w3id.org/xapi/video/templates#seeked", "https://w3id.org/xapi/video/templates#volumechange", "https://w3id.org/xapi/video/templates#screenchange", "https://w3id.org/xapi/video/templates#completed"]
```

Comment:
You probably don't want general restrictions here (or anywhere else). Since it matches literally every statement using this profile, if it is used it is superfluous to list anything else in an alternates -- they might as well not be there, and all guidance from a pattern is lost.